src/go/rpk/pkg/cli/cloud/cluster/secret/create.go

```diff
@@ -0,0 +1,77 @@
+package secret
+
+import (
+	"encoding/base64"
+	"strings"
+	"unicode"
+
+	v1alpha2 "buf.build/gen/go/redpandadata/dataplane/protocolbuffers/go/redpanda/api/dataplane/v1alpha2"
+	connect "connectrpc.com/connect"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/publicapi"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewCreateCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var secretName, secretValue string
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a new secret",
+		Long:  "Create a new secret for your Redpanda cloud cluster",
+		Run: func(cmd *cobra.Command, _ []string) {
+			// Validate secretName for non-lowercase letters
+			for _, r := range secretName {
+				if !unicode.IsLower(r) && unicode.IsLetter(r) {
+					out.Die("secret name must be lowercase")
```

Comment:
Is `foo_bar` allowed, or `foo^%$#bar`? This logic won't care about non-unicode letters. Please consider abstracting the validation function and having tests for it if you see fit.